packages/paste-website/src/components/paste-mdx-provider/index.tsx

```diff
@@ -92,7 +92,7 @@ const MDXPoviderComponents = {
     <Separator {...props} orientation="horizontal" verticalSpacing="space100" />
   ),
   a: (props: AnchorProps): React.ReactElement => <MdxLink {...props} />, // eslint-disable-line jsx-a11y/anchor-has-content
-  img: (props: React.ComponentProps<'img'>): React.ReactElement => <img {...props} />, // eslint-disable-line jsx-a11y/alt-text
+  img: (props: React.ComponentProps<'img'>): React.ReactElement => <img style={{maxWidth: '100%'}} {...props} />, // eslint-disable-line jsx-a11y/alt-text
```

Comment:
Note: large images embedded into markdown were overflowing the right side table of contents. Setting this max-width forces them to respect their content area and resize correctly.